src/python/pants/notes/1.21.x.rst

```diff
@@ -0,0 +1,214 @@
+1.21.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.21.x`` ``stable`` series.
+
+1.21.0rc0 (09/23/2019)
+----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* Add "did you mean" functionality to runtime Param mismatches. (#8315)
+  `PR #8315 <https://github.com/pantsbuild/pants/pull/8315>`_
+
+Bugfixes
+~~~~~~~~
+* Fix support for running pants from source in other repositories. (#8297)
```

Comment:
Ditto on moving this to tooling. It only impacts people developing Pants, not normal end users